src/content-tags-drawer/data/apiHooks.jsx

```diff
@@ -142,5 +144,30 @@ export const useContentTaxonomyTagsUpdater = (contentId, taxonomyId) => {
     onSettled: () => {
       queryClient.invalidateQueries({ queryKey: ['contentTaxonomyTags', contentId] });
     },
+    onSuccess: () => {
+      if (window.top != null) {
+        // Sends content tags to the parent page if is called from a iframe.
+        getContentTaxonomyTagsData(contentId).then((data) => {
+          const messageJson = {
+            contentId,
+            ...data,
+          };
+          window.top?.postMessage(
+            `[Manage tags drawer] Tags updated: ${JSON.stringify(messageJson)}`,
```

Comment:
I like this new message format, it might make sense to also change the `closeManageTagsDrawer` message to match the same format to keep things consistent?

Reply:
I think it would be better to add it when necessary, when it is used. Also, as [Braden](https://github.com/openedx/frontend-app-course-authoring/pull/800#discussion_r1459892950) says, these messages are temporary.

Reply:
I'm not sure why this message data is being passed as a string and then processed. You can simply pass JSON data. This is used in a couple of places and it generally follows the format of: `window.top?.postMessage({ type: 'authoring.events.tags.updated', payload: data })`

Then you can use code like [this](https://github.com/openedx/frontend-app-learning/blob/55e2332b0035c183104ad54814456e9d7a1e8fe0/src/courseware/course/sidebar/common/SidebarBase.jsx#L27-L33) to handle it on the other side.